en/ui-system/components/engine/auto-layout.md

```diff
@@ -2,40 +2,61 @@
 
 The layout component can be mounted to any node, making the node into a container with the auto layout function. The so-called auto layout container can automatically array the child nodes according to certain rules and adjust the container type nodes of its own size according to the sum total of bounding boxes of the node content.
```

Comment:
The layout component can be mounted to any node
-->
The __Layout__ component can be attached to any __Node__